### ISSUE TITLE
Let NVDA decide if objects should be included, instead of filtering out objects, and allow the user to only move the navigator object to the result

### ISSUE THREAD

**Issue by Emil-18** [doubletaponair/NVDAItemChooser#2]
Hi. I recommend checking if the simple review mode setting is active, and if so, exclude all objects that has presentationType not set to content. When simple review mode is not active, you shouldn't do any filtering at all. In this way, you let NVDA and other add-ons decide what objects should be included. Filtering based on states, roles and action count as you do now, isn't ideal. What if the user wants to locate and move to for example a piece of text?
Could you also allow the user to move only the navigator object to the result?
